data.json

```diff
@@ -1,4 +1,13 @@
 [
+   {
+    "name": "7PHP",
+    "difficulty": "easy",
+    "description": "If you are a fan of 7PHP, just send email at 'blogcontact@7php.com' and receive your free stickers.",
```

Comment:
I feel like the email shouldn't be in the description. It exposes it more to bots and it's easy to find anyway

Reply:
ill remove the email immediately.